README.md

```diff
@@ -0,0 +1,161 @@
+<a id="readme-top"></a>
+# Link to back-end
+[Back-end repo](https://github.com/theresetuyi/hello-rails-back-end/tree/rails-react)
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 React Rails practice I ](#-capstone_project_2_api-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [Live demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 React Rails practice I <a id="about-project"></a>
+
+Small project created with the purpose of learning how to implement React inside a Ruby on Rails application
+
+## 🛠 Built With <a id="built-with"></a>
+
+### Tech Stack <a id="tech-stack"></a>
+
+- Ruby on Rails
+- React
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">Postgresql</a></li>
+  </ul>
+</details>
```

Comment:
For a professional repository, it is important that our links are working correctly.

Every link should navigate to the expected `URLs` please correct this 💪😊 

![image](https://github.com/theresetuyi/hello-rails-back-end/assets/78251927/3171bc82-1809-4ff9-a560-7c16a46af283)